docs/virtualkeyboard/index.html

```diff
@@ -160,19 +178,37 @@ <h2>The <dfn>VirtualKeyboard</dfn> Interface</h2>
                 A few people have <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/415">expressed concerns</a> around {{VirtualKeyboard/show()}} and {{VirtualKeyboard/hide()}} methods not being promise-based, but we believe it is a better design for web developers to use {{VirtualKeyboard/ongeometrychange}} event since it is fired when VK visibility changes, making the return values unnecessary.
               </p>
             <dt>
-              <dfn>overlaysContent</dfn>
+              <p>{{VirtualKeyboard/overlaysContent}}</p>
             </dt>
             <dd>
               <p>
-                When this attribute is set to `true`, a user agent MUST NOT resize its layout viewport or visual viewport.
+                The {{VirtualKeyboard/overlaysContent}} getter steps are to return [=this=]'s {{VirtualKeyboard/overlaysContent}}.
+              </p>
+              <p>
+                The {{VirtualKeyboard/overlaysContent}} setter steps are to set [=this=]'s {{VirtualKeyboard/overlaysContent}} to the given value.
               </p>
             </dd>
             <dt>
-              <dfn>boundingRect</dfn>
+              <p>{{VirtualKeyboard/boundingRect}}</p>
             </dt>
             <dd>
             <p>
-                The attribute reports the intersection of the VK with the layout viewport in client coordinates.
+                The attribute reports the intersection of the VK with the layout viewport in client coordinates. It is set when {{VirtualKeyboard/ongeometrychange}} event is fired.
+                Setter steps are as follows:
+                <ol>
+                    <li>
+                        Let `osk` be the on-screen keyboard rectangle in CSS pixels that is provided by the OS when the VK visibility changes.
+                    </li>
+                    <li>
+                        Let `lv` be the layout viewport's bounds in CSS pixels.
+                    </li>
+                    <li>
+                        Set [=this=]'s {{VirtualKeyboard/boundingRect}} to be the intersection of `lv` and `osk` in CSS pixels.
```

Comment:
This algorithm feels like a getter, rather than a setter. If so, it should return a `DOMRect` but it doesn't create one. I'd expect it to do so while initializing it to the right values.

Reply:
Added a `DOMRect` object to store the intersection of the rectangles.